hadoop-hdds/docs/content/beyond/RunningWithHDFS.zh.md

```diff
@@ -0,0 +1,64 @@
+---
+title: 与 HDFS 并存运行
+linktitle: Runing with HDFS
+weight: 1
+summary: Ozone 能够与 HDFS 并存运行，本页介绍如何将 Ozone 部署到已有的 HDFS 集群上。
+---
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+Ozone 支持与 HDFS 并存工作，所以在已有的 HDFS 集群上部署 Ozone 比较容易。
+
+Ozone 的容器管理组件可以在 HDFS 数据节点上运行，以插件的形式或是独立运行，下文介绍插件运行的方法。
+
+为了启用 Ozone，你需要定义服务插件实现类。
+
+<div class="alert alert-warning" role="alert">
+<b>重要</b>：因为插件在 HDFS 数据节点启动过程中被激活，服务插件实现类的定义需要添加到 <b>hdfs-site.xml</b> 中。
+</div>
+
+{{< highlight xml >}}
+<property>
+   <name>dfs.datanode.plugins</name>
+   <value>org.apache.hadoop.ozone.HddsDatanodeService</value>
+</property>
+{{< /highlight >}}
+
+此外还需要将 /opt/ozone/share/ozone/lib/ 路径下的 jar 包添加到 classpath 下： 
+
+{{< highlight bash >}}
+export HADOOP_CLASSPATH=/opt/ozone/share/ozone/lib/*.jar
+{{< /highlight >}}
+
+
+
+让 ozone 随 HDFS 一同启动的步骤为：
```

Comment:
I know that original doc is "ozone", but I think it is better to fix it to "Ozone" for consistence in this part.
```suggestion
让 Ozone 随 HDFS 一同启动的步骤为：
```